requirements.txt

```diff
@@ -1,2 +1,3 @@
+PyYAML==5.4.1
```

Comment:
too late for this now, but I wonder if we can switch to https://pypi.org/project/strictyaml/

Reply:
I'll test strictyaml with craft-parts-orig parallel and refactor craft-parts if everything goes well.